### ISSUE TITLE
epgsql_codec_json how ?

### ISSUE THREAD

**Issue by heiheshang** [zotonic/zotonic#2551]
my project has jsonb type columns, how can I set the options for the epgsql_codec_json connection? zotonic sets a limited set of options when establishing a connection, i am having problems with json

**Comment by mworrell** [comment]
@mmzeeman you are using JSON in the db, can you pick this up?

**Comment by mmzeeman** [comment]
Yes, we use JSON with 0.x.. I had to upgrade the epgsql driver to make it work.

**Comment by heiheshang** [comment]
if no options are used then the json driver will be returned as a string

**Comment by mmzeeman** [comment]
In 0.x the encoder/decoder is passed here: https://github.com/zotonic/zotonic/blob/0.x/src/db/z_db_pgsql.erl#L439

**Comment by heiheshang** [comment]
it doesn't work for me, binary is returned

**Comment by heiheshang** [comment]
it looks like I have an old version, I'll try to update

**Comment by mmzeeman** [comment]
Are you using master or 0.x. I see that the PR for this is still open. I have to check that later.

**Comment by mworrell** [comment]
If it is ready for master, then you can merge it.

**Comment by heiheshang** [comment]
i am using 0.x
